engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/fenced-pages.service.js

```diff
@@ -15,6 +15,7 @@ angular.module('Bastion.organizations').service('FencedPages',
             'subscriptions',
             'gpg-keys',
             'sync-plans',
+            'sync-plan',
```

Comment:
I realized this while working on this PR, I just filed http://projects.theforeman.org/issues/17559 to make sure we go back and check the already converted pages.